src/main/java/WayofTime/bloodmagic/meteor/Meteor.java

```diff
@@ -29,6 +30,7 @@ public Meteor(ItemStack catalystStack, List<MeteorComponent> components, float e
         for (MeteorComponent component : components)
             weight += component.getWeight();
         this.maxWeight = weight;
+        this.cost = 1000000;
```

Comment:
GSON initializes classes with reflection and not the constructor. The default definition should be set at the field declaration.